templates/ceph/consulting.html

```diff
@@ -0,0 +1,216 @@
+				</div>
+			</li>
+			<li class="p-matrix__item">
+				<div class="p-matrix__content">
+				<h3 class="p-matrix__title">Day-2 automation</h3>
+				<p class="p-matrix__desc">Charmed Ceph provides automation around common tasks, such as cluster scaling, integration and Ceph upgrades.</p>
+				</div>
+			</li>
+			<li class="p-matrix__item">
+				<div class="p-matrix__content">
+				<h3 class="p-matrix__title">Optional training</h3>
+				<p class="p-matrix__desc">If required we can deliver Ceph <a href="/training">training courses</a> to your team to familiarise them with Charmed Ceph.</p>
+				</div>
+			</li>
+		</ul>
+  </div>
+</section>
+<section class="p-strip">
+	<div class="u-fixed-width">
+		<h2>Ceph Cluster Build pricing</h2>
+	</div>
+		{% include "shared/pricing/_ceph-cluster-build-pricing.html" %}
+</section>
+<section class="p-strip--light">
+	<div class="row">
+	  <div class="col-8">
+	  	<h2> Companies using Ceph </h2>
+	  	<p>There are multiple users of Ceph across a broad range of industries, from academia to telecommunications and cloud service providers. Ceph is particularly favoured for its flexibility, scalability, and robustness.</p>
+	  </div>  
+	</div>
+	<div class="u-fixed-width">
+		<div class="p-logo-section">
+			<p class="p-logo-section__title u-align--center">
+				NOTABLE CEPH USERS
+			</p>
+			<div class="p-logo-section__items">
+				<div class="p-logo-section__item u-align--center">
+					{{ image (
+						url="https://assets.ubuntu.com/v1/9131d1b3-CERN_logo.svg",
+						alt="CERN",
+						height="60",
+						width="60",
+						hi_def=True,
+						attrs={"class": "p-logo-section__logo"},
+						loading="lazy",
+						) | safe
+					}}
+				</div>
+				<div class="p-logo-section__item">
+					{{ image (
+						url="https://assets.ubuntu.com/v1/d199354d-2018-logo-deutsche-telekom.svg",
+						alt="Deutsche Telekom",
+						height="145",
+						width="145",
+						hi_def=True,
+						attrs={"class": "p-logo-section__logo"},
+						loading="lazy",
+						) | safe
+					}}
+				</div>
+				<div class="p-logo-section__item">
+					{{ image (
+						url="https://assets.ubuntu.com/v1/9ae8ff55-2018-logo-bloomberg.svg",
+						alt="Bloomberg",
+						width="145",
+						height="145",
+						hi_def=True,
+						attrs={"class": "p-logo-section__logo"},
+						loading="lazy",
+						) | safe
+					}}
+				</div>
+				<div class="p-logo-section__item">
+					{{
+						image(
+						url="https://assets.ubuntu.com/v1/80ab9b3c-2018-logo-cisco.svg",
+						alt="Cisco",
+						height="145",
+						width="145",
+						hi_def=True,
+						attrs={"class": "p-logo-section__logo"},
+						loading="lazy",
+						) | safe
+					}}
+				</div>
+				<div class="p-logo-section__item">
+				{{
+					image(
+					url="https://assets.ubuntu.com/v1/5029f949-Dream-Host-logo.png",
+					alt="Dream Host",
+					height="46",
```

Comment:
This image has a white background on the grey strip which doesn't look great, you could upload a transparent background one to the asset server to use

Reply:
Thanks Beth! :) Got a new logo!